cockatrice/src/tab_deck_editor.cpp

```diff
@@ -700,7 +700,9 @@ void TabDeckEditor::updateCardInfoRight(const QModelIndex &current, const QModel
 
 void TabDeckEditor::updateSearch(const QString &search)
 {
-    databaseDisplayModel->setCardName(search);
+    QString sanitizedInput = QString(search);
+    sanitizedInput.replace("’", "'");
+    databaseDisplayModel->setCardName(sanitizedInput);
```

Comment:
What do you think about putting this logic inside `setCardName`?

Reply:
I wasn't sure how many other modules relied on `setCardName`, and that maybe the distinction between the curly apostrophe and the ASCII apostrophe was meaningful.

I did look and it seems like `dlg_crate_token.cpp:180` does use it, via a signal sent from a text box containing the default text "Token", however the function containing this statement doesn't appear to be used anywhere else in the program.

In this case, I think that it's safe to put this logic in `serCardName`.